mixer/pkg/il/testing/tests.go

```diff
@@ -305,6 +305,50 @@ end`,
 		R:    true,
 		conf: exprEvalAttrs,
 	},
+	// use request.header (without "s" at the end), as it is the one in the original attributeset.
```

Comment:
should we update the testing set to match the actual vocab (`request.headers`) so as not to sow confusion?

Reply:
Added 0.6 attribute set as a separate set.

I'd rather not pollute the previous tests that I copied over from previous expression tests.